Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,209 @@
 ﻿using System;
+using System.Text;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        private const string LeftArrow = "\u2190";
+        private const string UpwardArrow = "\u2191";
+        private const string RightArrow = "\u2192";
+        private const string DownArrow = "\u2193";
+
+        private const char LeftKey = 'A';
+        private const char UpwardKey = 'W';
+        private const char RightKey = 'D';
+        private const char DownKey = 'S';
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Part 1 - Convert with 2 conversion modes: From decimal to binary, From binary to decimal.
+            ConvertNumberSystem();
+
+            //Part 2 - Use your WASD (both capital and non capital) keys to print arrows.
+            ConvertLettersToArrows();
+        }
+
+        private static void ConvertNumberSystem()
+        {
+            var mode = GetConversionMode();
+            var number = GetNumber(mode);
+            var convertedNumber = (mode == 1) ? ConvertDecimalToBinary(int.Parse(number)) : ConvertBinaryToDecimal(number).ToString(); ;
```

Comment:
1 serves a bit as a magic number here. We will tackle this later on but you could have resolved that by using another constant variable. This is a minor issue though. :)